src/components/productCard/index.tsx

```diff
@@ -0,0 +1,32 @@
+import AddToCartIcon from 'assets/icons/addToCartIcon.tsx';
+import {
+  ProductImageWrapper,
+  ProductInfo,
+  ProductName,
+  ProductPrice,
+  VendorName,
+} from 'components/productCard/styles.ts';
+import Carousel from 'components/carousel';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
commented code

Reply:
used to ignore unused variable until we fetch data from server